create documentation for operations using the new Beagle dsl (#767)
ZupIT/beagle-docs: content/pt/resources/dsl-operations/logic-operator.md

---
title: Operadores lógicos
weight: 365

> description: 'Nesta seção, você encontra descrição completa dos Operadores Lógicos.'

Reviewer: ```suggestion
description: 'Nesta seção, você encontra descrição completa dos operadores lógicos.'
```

---

---

## Quais são? 

> 

Reviewer: Veja abaixo quais são os operadores lógicos:

| Operadores | Ação |
| :---: | :--- | :--- |
| condition\(\) | Operador de condição que avalia dois elementos e verifica qual é a condição verdadeira. |
| not\(\) | Operador de negação que tem como entrada apenas um valor, e sua função é simplesmente inverter esse valor. |
| and\(\) | Operador que recebe duas entradas e resulta em um valor **VERDADEIRO** se os dois valores de entrada da operação forem **VERDADEIROS**, caso contrário o resultado é **FALSO**. |
| or\(\) | Operador que recebe duas entradas E resulta em um valor **VERDADEIRO** se ao menos **UM** dos dois valores de entrada da operação for **VERDADEIRO**, caso contrário o resultado é **FALSO**. |

### Exemplo

> Aqui temos o exemplo de uma tela que utiliza duas operações lógicas, `condition` e `or`, onde caso o resultado da condição do `or` for verdadeira o texto do componente `Text` se torna **true**, caso falso ele atribui ao texto o valor de **false.**

Reviewer: ```suggestion
Este exemplo é uma tela que utiliza duas operações lógicas `**condition**` e `**or**`; 
Há duas possibilidades no resultado: 
1. Caso o resultado da condição `or` for verdadeira o texto do componente `Text` se torna **true**.
2. Se for falso ele atribui ao texto o valor de **false.**
```

---
title: Operadores lógicos
weight: 365
description: 'Nesta seção, você encontra descrição completa dos operadores lógicos.'
---

---

## Quais são? 

> 

Reviewer 2: ```suggestion

Veja abaixo quais são os operadores lógicos:
```
